PROCESS.md

```diff
@@ -0,0 +1,163 @@
+post SHOULD be prepared quarterly to give visibility to any changes that were
+merged in the last quarter.
+
+## The Stability Model
+Every feature in the spec has a release status. It's either evolving, stable, or
+deprecated. Flags are used to show the status a feature is in. If a feature
+doesn't have a flag, it's considered stable. If it has a year flag (such as
+`2024`) that's the year the feature reached stability. The other flags indicate
+a feature that is not yet stable or is deprecated.
+
+### Champions
+Any feature that is evolving SHOULD have a champion. The champion is responsible
+for shepherding the feature through the process to into stable status. The
+champion doesn't need to do all the work, they're just responsible for making
+sure the work gets done. The champion role MAY be shared by more than one
+person. An evolving feature without a champion SHOULD be removed from the spec.
+
+### Proposals
+New features start out as proposals. Proposals are informal GitHub Issues or
+Discussions and can be submitted by anyone.
+
+All proposals MUST be backward compatible with the current stable specification.
+Features that could be problematic for forward compatibility are highly
+discouraged, but may be considered if the feature is optional and disabled by
+default.
+
+**Promotion Checklist:**
+[] Consensus among JSON Schema Org core contributors that the proposal is worth
+  pursuing
+[] The proposal has a champion
+[] A PR for an entry for in the `STAGE-0` registry has been approved
+
+### STAGE-0
+`STAGE-0` features are tracked in the `STAGE-0` Registry on the website. This
+registry lists all `STAGE-0` features. Each entry SHOULD include the following
+checklist as well as an informal description with at least one example and any
+relevant links.
+
+This stage requires at least two implementations to implement the feature. The
+purpose of these implementations is to identify any potential implementation
+issues that might lead to low adoption such as difficulty of implementation or
+performance implications. Therefore, these implementations don't need to be
+publicly available or have real world use at this point.
+
+**Promotion Checklist:**
+[] There is general consensus among the core contributors for adding the feature
+  to the spec
+[] Tests are available in the test suite
+[] Two implementations have implemented the feature and pass the tests suite
+[] A blog post has been published introducing the feature
+[] A PR for the spec has been approved that completely specifies the feature and
+  removes the feature from the `STAGE-0` registry.
+
+### STAGE-1
+`STAGE-1` features are included to the specification, but flagged as `STAGE-1`
+and are not subject to the compatibility requirements of "stable" features. They
+can be added at any time as long as it meets all criteria for `STAGE-1` status.
+
+At this stage, the feature should be seeing real world use. These features MAY
```

Comment:
I'm curious about how we will guage/measure this real world use.  Wouldn't we be reliant solely on volunteered feedback?

Reply:
I think in general, maybe.  But that may also be skewed toward more popular implementations that may or may not support a feature.

I like your answer below about including Bowtie and extracting metrics from there.  It shows support for a feature, but not necessarily usage.

Reply:
I don't think this is something that we have to resolve now.

Reply:
This is a tough one. I think it will always be largely a judgement call, but we should definitely be looking for more and better ways we can gather data we can use to make those calls.